scintellometry/phasing/GMRTNaming.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+phasing_dir=os.path.dirname(os.path.realpath(__file__))
```

Comment:
I wonder if it wouldn't be handier to just create a directory directly (in `io.gmrt`):
```
nodes_and_voltages = {
    'C00R': (0, 1),
    'C00L': (8, 1),
...
```
Or, perhaps still read in the file once and create the dictionary. 

But it doesn't matter all that much. However, if you stick to a function, I would suggest in any case returning both node and voltage in one go, and raising an exception (probably `ValueError`) here rather than in the parser.